plat/mediatek/common/drivers/uart/8250_console.S

```diff
@@ -89,10 +98,40 @@ func console_core_init
 core_init_fail:
 	mov	w0, wzr
 	ret
-endfunc console_core_init
+endfunc console_8250_core_init
+
+#if MULTI_CONSOLE_API
```

Comment:
As @jwerner-chromium mentions, we intend to remove the old console API soon.

Reply:
Since this PR is for platform mt8183 initialization,
could removing old console API of old MTK platforms be done by another PR?

Reply:
Well, you're adding MULTI_CONSOLE_API support to the driver here, so it would be nicer if you could just switch all the way right now. Shouldn't it just need a two-line change each to make the two old SoCs conform to it?

On the other hand, if you follow my suggestion above to switch to the default 16550 driver rather than rolling your own, you may be able to reuse the legacy console support from there for now.

Reply:
> As part of the migration to trustedfirmware.org [ARM-software/tf-issues#681](https://github.com/ARM-software/tf-issues/issues/681) we have switched the main development repository to Gerrit (https://review.trustedfirmware.org/). It is unfortunate that the switch has happened while your PR was being reviewed, but we don't intend to merge more patches through GitHub to avoid confusion.
> 
> Could you please follow the instructions in https://github.com/ARM-software/arm-trusted-firmware/blob/master/contributing.rst to continue with the review? We're sorry for the inconvenience.

Have just updated the commit according to your suggestion and rebase forked branch on latest master branch.
Then, I will push the commit and continue with review on https://review.trustedfirmware.org/.
Thanks.